src/main/java/org/jfrog/bamboo/context/PublishBuildInfoContext.java

```diff
@@ -0,0 +1,64 @@
+package org.jfrog.bamboo.context;
+
+import com.google.common.collect.Sets;
+import java.util.Set;
+import java.util.Map;
+
+import static org.jfrog.bamboo.context.AbstractBuildContext.CAPTURE_BUILD_INFO;
+
+/**
+ * @author Alexei Vainshtein
+ */
+public class PublishBuildInfoContext implements ArtifactoryContextInterface {
+
+    public static final String SERVER_ID_PARAM = "artifactory.artifactoryPublishBuildInfo.artifactoryServerId";
+    public static final String USERNAME = "artifactory.artifactoryPublishBuildInfo.username";
+    public static final String PASSWORD = "artifactory.artifactoryPublishBuildInfo.password";
```

Comment:
I suggest changing the prefix of the above 3 constants from:
artifactory.artifactoryPublishBuildInfo.
to:
artifactory.task.publishBuildInfo.

Reply:
Done